pkg/controllers/webhook/utils.go

```diff
@@ -19,20 +19,31 @@ import (
 type webhook struct {
 	maxWebhookTimeout int32
 	failurePolicy     admissionregistrationv1.FailurePolicyType
-	rules             map[schema.GroupVersion]sets.Set[string]
+	rules             map[GroupVersionScope]sets.Set[string]
+}
+
+type GroupVersionScope struct {
```

Comment:
Do we need to make it public?

Reply:
@fhopfensperger - what's your thought on this?

Reply:
@realshuting just to clarify, you're asking me if we want to make `rules` publicly available, right? 
If so, I don't have a strong opinion on that, i use the same visibility as other attributes.

Reply:
I can make them all private or all public, same goes for `ScopeType`. If you can tell me how you want it, I will change it.

Reply:
@realshuting  I have just updated the `Rules` and `ScopeType` attributes to make them private.